staging/src/k8s.io/apiextensions-apiserver/test/integration/testserver/resources.go

```diff
@@ -37,6 +37,26 @@ const (
 	noxuInstanceNum int64 = 9223372036854775807
 )
 
+// New generates a CRD with random name to avoid name conflict in e2e tests
+func NewNoxuCustomResourceDefinitionForAdmissionWebhookTest(scope apiextensionsv1beta1.ResourceScope) *apiextensionsv1beta1.CustomResourceDefinition {
```

Comment:
Noxu?

Reply:
@deads2k's hobby is to invent names and confuse us.